src/InputContainer.test.jsx

```diff
@@ -0,0 +1,47 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import { useSelector, useDispatch } from 'react-redux';
+
+import InputContainer from './InputContainer';
+
+jest.mock('react-redux');
+
+describe('InputContainer', () => {
+  useSelector.mockImplementation((selector) => selector({
+    taskTitle: 'New Title',
+  }));
+
+  const dispatch = jest.fn();
+  useDispatch.mockImplementation(() => dispatch);
+
+  function customRender() {
+    return render((
+      <InputContainer />
+    ));
+  }
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders to do list', () => {
+    const { queryByText, getByDisplayValue } = customRender();
+
+    expect(queryByText(/추가/)).not.toBeNull();
+    expect(getByDisplayValue(/New Title/)).not.toBeNull();
```

Comment:
queryBy 로 수정하면 좋을거 같아요 :)
```suggestion
    expect(queryByDisplayValue(/New Title/)).not.toBeNull();
```

Reply:
넵!